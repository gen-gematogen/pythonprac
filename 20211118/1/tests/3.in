@objcount
class mamba:
    def __del__(self):
        print("Mamba destroyed!")
c, d, e = mamba(), mamba(), mamba()
print(mamba.counter)
c = 100500
print(mamba.counter)
