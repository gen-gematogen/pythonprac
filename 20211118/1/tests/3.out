3
Mamba destroyed!
2
Mamba destroyed!
Mamba destroyed!
